python/scripts/rfi_receiver/rfi_receiver.py

```diff
@@ -359,27 +360,35 @@ def data_listener(thread_id):
             InitialKotekanConnection = True
             logger.info("Connected to Kotekan")
 
+        # A packet is received roughly every ~0.126s
         if packet != "":
 
+            # Print every ~6s
             if packetCounter % (50 * len(stream_dict) + 1) == 0:
                 logger.info(
                     "Thread id %d: Receiving Packets from %d Streams"
                     % (thread_id, len(stream_dict))
                 )
```

Comment:
I'm not sure this message adds any value, the second message indirectly gives the number of streams by way of the list - or you could just include the number of streams in the second message.